docs/examples/kymo_markers/kymo_markers.rst

```diff
@@ -0,0 +1,206 @@
+
+Kymograph with markers
+======================
+
+.. only:: html
+
+    :nbexport:`Download this page as a Jupyter notebook <self>`
+
+.. _kymo_markers:
+
+Convert kymograph coordinates to basepairs using markers
+--------------------------------------------------------
+
+In this notebook we use two red markers with known coordinates on a kymograph to convert coordinates from micrometers to base pair.
+The workflow is as follows:
+
+- Determine the location of the red markers using a peak detection algorithm on the red channel 
+- Use the known coordinates of the markers to convert coordinates to basepairs
+
+We then compare the binding profile, with base pair coordinates, to the expected target sites for the protein.
+
+This Notebook requires the `peakutils` package. Run the following cell to make sure it is installed:
+
+Download the data file
+----------------------
+
+The kymograph is stored on zenodo.org.
+We can download the data directly from Zenodo using the function :func:`~lumicks.pylake.download_from_doi`.
+The data will be stored in the folder called `"test_data"`::
+
+    filenames = lk.download_from_doi("10.5281/zenodo.7729525", "test_data")
+
+Plot the kymograph
+------------------
+
+Before starting the analysis on the high frequency data, let's look at the complete kymograph::
+
+    file = lk.File("test_data/kymo.h5")
+    kymo = file.kymos["16"]
+    plt.figure()
+    kymo.plot("rgb", adjustment=lk.ColorAdjustment(0, 98, mode="percentile"))
+
+.. image:: kymo_complete.png
+
+The above image shows the full kymograph. 
+
+Markers and target sites
+------------------------
+
+Provide the binding coordinates of the red markers and the two expected target sites for the gren protein, on lambda DNA::
+
+    marker1 = 33.786  # The location of the markers in kbp
+    marker2 = 44.826
+    dna_kbp = 48.502  # length of DNA in kbp
+    target_sites = [18.084, 30.533]  # Target binding sites
```

Comment:
Just out of curiosity, why'd you choose to put the sites in an array and the markers in separate floats?

Reply:
Originally I had a file with 10 target sites. It does make sense to add the markers to an array too.